packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -760,9 +743,7 @@ export default class CallDiagnosticMetrics extends StatelessWebexPlugin {
     this.logger.log(
       CALL_DIAGNOSTIC_LOG_IDENTIFIER,
       'CallDiagnosticMetrics: @submitClientEvent. Submit Client Event CA event.',
-      `name: ${name}`,
-      `payload: ${JSON.stringify(payload)}`,
-      `options: ${JSON.stringify(options)}`
+      `name: ${name}`
```

Comment:
suggestion: maybe it would be a good idea to log here any `payload.errors` or if `payload.canProceed` is false?

Reply:
there's a log above with 
CallDiagnosticMetrics: @prepareClientEvent. Generated errors:',
        `generatedError: ${JSON.stringify(generatedError)}`
        
        So you'll see it there